src/useInterApplicationBusPublish.ts

```diff
@@ -11,7 +11,7 @@ export default <T>(topic: string, message: T) => {
     };
 
     useEffect(() => {
-        const fin: any = window;
+        const fin: any = (window as any).fin;
```

Comment:
Rather than casting to `any`, could we add `fin` to the Window interface?

e.g. create a` globals.d.ts` file in `src` with:
```
import {Fin} from "openfin/_v2/main";

declare global {
    // tslint:disable-next-line:interface-name
    interface Window {
        fin: Fin;
    }
}
```